### PR TITLE
Improve code coverage

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:
Adds basic tests for:
- rbac generation for operator

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by acardace** [comment]
/lgtm
nice work :)

File: pkg/virt-operator/resource/generate/rbac/operator_test.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+var _ = Describe("RBAC", func() {
+
+	Context("GetAllOperator", func() {
+
+		forOperator := GetAllOperator("default")
```
Comment (ashleyschuett) [inline pkg/virt-operator/resource/generate/rbac/operator_test.go:36]: Let's pull out "default" to a const.

**Comment by ashleyschuett** [comment]
Looks good other than one nit. Nice work!

**Comment by dhiller** [comment]
@ashleyschuett @acardace thanks for the review, addressed review comments. PTAL, thanks!

**Review by acardace — APPROVED** [review]

**Pull request merged**